code/modules/client/preferences.dm

```diff
@@ -534,6 +535,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if(jobban_isbanned(user, rank))
 				HTML += "<font color=red>[rank]</font></td><td><a href='?_src_=prefs;jobbancheck=[rank]'> BANNED</a></td></tr>"
 				continue
+			var/required_playtime_remaining = job.required_playtime_remaining(user.client)
+			if(required_playtime_remaining)
+				HTML += "<font color=red>[rank]<//font></td><td><font color=red> \[ [get_exp_format(required_playtime_remaining)] as [job.get_exp_req_type()] \] </font></td></tr>"

```

Comment:
```HTML += "<tr bgcolor='[job.selection_color]'><td width='60%' align='right'>"```
Line 532.

The only thing wrong with this code is that ```HTML``` is a string and not a list, so it'll perform shit + the //font tag.